### PR TITLE
Added prefers-reduced-motion media query

### PR DESCRIPTION
**Pull request by yepninja**

### CODE REVIEW

File: cssdb.json [hunk 1]
```diff
@@ -526,6 +526,15 @@
       }
     ]
   },
+  {
+    "id": "prefers-reduced-motion",
```
Comment (jonathantneal) [inline cssdb.json]: Could we append `-query` to media query features? We do the same for functions and properties and pseudo classes.

```suggestion
    "id": "prefers-reduced-motion-query",
```

File: cssdb.json [hunk 2]
```diff
@@ -526,6 +526,15 @@
...
+    "title": "`prefers-reduced-motion` Media Query",
+    "description": "Media query based on a user preference for preferring reduced motion (animation, etc)",
```
Comment (jonathantneal) [inline cssdb.json]: Could we describe this as the following?

```suggestion
    "description": "A media query to detect a user’s desire for less animation and general motion on the page",
```

Comment (yepninja) [reply cssdb.json]: Thank you for helping with texts!

**Comment by yepninja** [comment]
Updated.

**Comment by jonathantneal** [comment]
Great work, @yepninja. Merged!

**Pull request merged**